src/app/view/perspectives/kanban/columns/column/kanban-column.component.ts

```diff
@@ -75,9 +94,15 @@ export class KanbanColumnComponent implements OnInit, OnChanges {
   @Input()
   public collections: Collection[];
 
+  @Input()
+  public linkTypes: LinkType[];
+
   @Input()
   public documents: DocumentModel[];
 
+  @Input()
+  public linkInstances: LinkInstance[];
```

Comment:
Why do we need to have instances of all these entities here if we already have an abstract structure such as `KanbanCard` which already carries those entities?

Reply:
It's quite same reason as above. I need to execute store action in this component, because it's necessary to scroll down to new post-it after successful action. These data are needed to correctly create document. The other solution would be to emit action to parent component and then call function on `ViewChild` from parent.